components/channel_notifications_modal/components/desktop_notification_section.test.jsx

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {NotificationLevels} from 'utils/constants';
+
+import DesktopNotificationSection from 'components/channel_notifications_modal/components/desktop_notification_section.jsx';
+import * as Utils from '../../../utils/utils';
+
+describe('components/channel_notifications_modal/DesktopNotificationSection', () => {
+    const baseProps = {
+        onChange: () => {}, //eslint-disable-line no-empty-function
```

Comment:
These can use `jest.fn()` instead of an arrow function which will get around ESLint